src/extensions/ignite.js

```diff
@@ -8,14 +8,14 @@ const findIgnitePluginsExt = require('./ignite/findIgnitePlugins')
 const addModuleExt = require('./ignite/addModule')
 const addAndroidPermissionExt = require('./ignite/addAndroidPermission')
 const removeModuleExt = require('./ignite/removeModule')
-const addScreenExamplesExt = require('./ignite/addScreenExamples') // Deprecated 3/2/17, Ignite 2 Beta
+const addScreenExamplesExt = require('./ignite/addScreenExamples') // Deprecated 3/2/17, Ignite CLI Beta
```

Comment:
There's a bunch of Deprecated 3/2/17 comments here. Were these require statements deprecated on 3/2/17?